src/CMakeLists.txt

```diff
@@ -789,8 +800,8 @@ endif (WITH_MGR)
 set(librados_config_srcs
   librados-config.cc)
 add_executable(librados-config ${librados_config_srcs})
-target_link_libraries(librados-config librados global ${BLKID_LIBRARIES} ${RDMA_LIBRARIES}
-  ${CMAKE_DL_LIBS})
+target_link_libraries(librados-config librados global ${BLKID_LIBRARIES} ${VERBS_LIBRARIES}
```

Comment:
i don't think we should link against RDMA_LIBRARIES or VERBS_LIBRARIES directly here. as this linkage is introduced by libcommon, and librados-config should not use symbols offered by these two libs.